lightning-invoice/src/lib.rs

```diff
@@ -18,11 +18,11 @@
 //! invoices and functions to create, encode and decode these. If you just want to use the standard
 //! en-/decoding functionality this should get you started:
 //!
-//!   * For parsing use `str::parse::<Invoice>(&self)` (see [`Invoice::from_str`])
+//!   * For parsing use `str::parse::<Bolt11Invoice>(&self)` (see [`Bolt11Invoice::from_str`])
 //!   * For constructing invoices use the [`InvoiceBuilder`]
```

Comment:
There's still two `InvoiceBuilder` structs, though I guess those aren't exposed in bindings anyway?

There's also:
- `InvoiceSignature`
- `InvoiceDescription`
not sure if those should be disambiguated

Reply:
> There's still two `InvoiceBuilder` structs, though I guess those aren't exposed in bindings anyway?
> 

Yeah, as noted in the PR description, those should probably be renamed but currently aren't exposed so I opted not to.

> There's also:
> 
> * `InvoiceSignature`
> * `InvoiceDescription`
>   not sure if those should be disambiguated

This is borderline. On one hand, they are part of a type that is already qualified so it seems redundant. On the other hand, they are public types. I'm somewhat indifferent here, so I can change those if you prefer.

Reply:
I thought bindings users lose module prefixes so if that's the case I'd be in favor of renaming. Nbd though.

Reply:
Are you referring to `InvoiceBuilder`? I was agreeing with that but just saying not now since they aren't currently exported in bindings. We can do it now if you prefer, though.

Reply:
Oh sorry, no I meant the description and signature.

Reply:
Gotcha. Renamed now.